### ISSUE TITLE
Add Table Headings

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1679]
Tables should have table headings to assist screen reader users in understanding what the table is for. Currently, tables in UOP lack headings
